pkg/controllers/clusterresourceplacement/resource_selector.go

```diff
@@ -230,43 +230,64 @@ func buildApplyOrderMap() map[string]int {
 	return ordering
 }
 
-// fetchClusterScopedResources retrieves the objects based on the selector.
-func (r *Reconciler) fetchClusterScopedResources(selector fleetv1beta1.ClusterResourceSelector, placeName string) ([]runtime.Object, error) {
-	klog.V(2).InfoS("start to fetch the cluster scoped resources by the selector", "selector", selector)
+// fetchResources retrieves the objects based on the selector.
+func (r *Reconciler) fetchResources(selector fleetv1beta1.ClusterResourceSelector, placementName types.NamespacedName) ([]runtime.Object, error) {
```

Comment:
TODO: I will rename the ClusterResourceSelector as it no longer select clusterResource only